src/cleanlab_tlm/utils/chat.py

```diff
@@ -97,14 +104,58 @@ def _format_tools_prompt(tools: list[dict[str, Any]], is_responses: bool = False
                     "parameters": tool["function"]["parameters"],
                 },
             }
-        else:  # responses format
+        # responses format
+        elif tool["type"] == "function":
             tool_dict = {
                 "type": "function",
                 "name": tool["name"],
                 "description": tool["description"],
                 "parameters": tool["parameters"],
                 "strict": tool.get("strict", True),
             }
+        elif tool["type"] == "file_search":
+            tool_dict = {
+                "type": "function",
+                "name": "file_search",
+                "description": "Search user-uploaded documents for relevant passages.",
+                "parameters": {
+                    "type": "object",
+                    "properties": {
+                        "queries": {
+                            "type": "array",
+                            "items": {"type": "string"},
+                            "description": "Search queries to run against the document index.",
+                        },
+                    },
+                    "required": ["queries"],
+                },
+            }
+        elif tool["type"] == "web_search_preview":
+            if importlib.util.find_spec("trafilatura"):
+                tool_dict = {
+                    "type": "function",
+                    "name": "web_search_call",
```

Comment:
"web_search_call" appears multiple times throughout this code. 

That should be defined at the top as a constant variable:

WEB_SEARCH_CALL = "web_search_call"

and then WEB_SEARCH_CALL should be used throughout the code instead (so we can update this easier in the future).

Same for other hardcoded strings that are used in multiple places.